gulpfile.js

```diff
@@ -20,7 +20,7 @@ gulp.task('lint-app', function () {
         .pipe(jshint.reporter('fail'));
 });
 
-gulp.task('uglify', ['lint-app'], function () {
```

Comment:
I didn't look carefully at the gulpfile, and didn't see that the task was intended to fail if linting errors were encountered :) I'll add it back in, it was an oversight on my part.